[BUG]  triggering sshguard blocking

### Title

 triggering sshguard blocking

### Platform

Website - Chrome

### Server Installation Method

Docker

### Version

2.1.0

### Troubleshooting

- [x] I have examined logs and tried to find the issue
- [x] I have reviewed opened and closed issues
- [x] I have tried restarting the application
- [x] I have read Known Issues (https://github.com/Termix-SSH/Support/issues/296)

### The Problem

start termix in docker ip  172.18.0.100 , host machine 192.168.1.165 , 
and try to access host machine with password authentication.

<img width="901" height="733" alt="Image" src="https://github.com/user-attachments/assets/4072e7d3-4cab-419e-b774-e577e4281109" />

after some time , host sshguard  blocking the acess:

<img width="1286" height="516" alt="Image" src="https://github.com/user-attachments/assets/4cf815bb-696f-47a4-b4b7-b1b872998687" />

don't known how could that happen , maybe some 'illegal' keep-alive request in terms of sshguard ?

i can modify /etc/sshgard/whitelist file to solve the problem ,  IMO this shouldn't happen;

### How to Reproduce

1. start termix 
2. access host machine (password authentication) with sshguard enabled
3. just let the browser  tab open , and do nothing 
4. the sshguard show that 'Attcack from ....' and finally get 'blocked'

### Additional Context

termix logs:

```
[7:16:18 AM] [INFO] [🖥️] Terminal session created after SSH ready [op:terminal_session_created,user:Snn9CG40W-qBMvnr7Zpna,host:3,session:f3e51ea5-2d2f-4fc6-af69-f07e607ef3d4]

[7:16:18 AM] [INFO] [🖥️] Creating shell [op:ssh_shell_start,host:3]

[7:16:19 AM] [SUCCESS] [🖥️] Terminal shell channel opened [op:terminal_shell_opened,user:Snn9CG40W-qBMvnr7Zpna,host:3,session:IdefLKyjAMVeu55LWR_v_]

[7:16:19 AM] [INFO] [🖥️] WebSocket attached to session [op:session_attach,user:Snn9CG40W-qBMvnr7Zpna,session:f3e51ea5-2d2f-4fc6-af69-f07e607ef3d4]

[7:16:19 AM] [INFO] [🖥️] Session ready for persistence [op:session_ready,user:Snn9CG40W-qBMvnr7Zpna,host:3,session:f3e51ea5-2d2f-4fc6-af69-f07e607ef3d4]

[7:16:20 AM] [INFO] [🖥️] Terminal WebSocket disconnected [op:terminal_ws_disconnect,user:Snn9CG40W-qBMvnr7Zpna,session:IdefLKyjAMVeu55LWR_v_]

[7:16:20 AM] [INFO] [🖥️] WebSocket detached from session [op:session_detach,user:Snn9CG40W-qBMvnr7Zpna,session:f3e51ea5-2d2f-4fc6-af69-f07e607ef3d4]

[7:16:29 AM] [INFO] [🖥️] Terminal WebSocket connection established [op:terminal_ws_connect,user:Snn9CG40W-qBMvnr7Zpna,session:IdefLKyjAMVeu55LWR_v_]

[7:16:29 AM] [INFO] [🖥️] Resolving SSH host configuration [op:terminal_host_resolve,user:Snn9CG40W-qBMvnr7Zpna,host:3,session:IdefLKyjAMVeu55LWR_v_]

[7:16:29 AM] [INFO] [🖥️] Initiating SSH connection [op:terminal_ssh_connect_attempt,user:Snn9CG40W-qBMvnr7Zpna,host:3,session:IdefLKyjAMVeu55LWR_v_]

[7:16:29 AM] [INFO] [🖥️] Host key verified successfully [op:host_key_verified,user:Snn9CG40W-qBMvnr7Zpna,host:3]

[7:16:29 AM] [SUCCESS] [🖥️] SSH connection established [op:terminal_ssh_connected,user:Snn9CG40W-qBMvnr7Zpna,host:3,session:IdefLKyjAMVeu55LWR_v_]

[7:16:29 AM] [INFO] [🖥️] Terminal session created [op:session_created,user:Snn9CG40W-qBMvnr7Zpna,host:3,session:0707aeb4-3615-4c54-a72c-20b04f140973]

[7:16:29 AM] [INFO] [🖥️] Terminal session created after SSH ready [op:terminal_session_created,user:Snn9CG40W-qBMvnr7Zpna,host:3,session:0707aeb4-3615-4c54-a72c-20b04f140973]

[7:16:29 AM] [INFO] [🖥️] Creating shell [op:ssh_shell_start,host:3]

[7:16:29 AM] [SUCCESS] [🖥️] Terminal shell channel opened [op:terminal_shell_opened,user:Snn9CG40W-qBMvnr7Zpna,host:3,session:IdefLKyjAMVeu55LWR_v_]

[7:16:29 AM] [INFO] [🖥️] WebSocket attached to session [op:session_attach,user:Snn9CG40W-qBMvnr7Zpna,session:0707aeb4-3615-4c54-a72c-20b04f140973]

[7:16:29 AM] [INFO] [🖥️] Session ready for persistence [op:session_ready,user:Snn9CG40W-qBMvnr7Zpna,host:3,session:0707aeb4-3615-4c54-a72c-20b04f140973]

[7:17:07 AM] [INFO] [🖥️] Terminal WebSocket disconnected [op:terminal_ws_disconnect,user:Snn9CG40W-qBMvnr7Zpna,session:IdefLKyjAMVeu55LWR_v_]

[7:17:07 AM] [INFO] [🖥️] WebSocket detached from session [op:session_detach,user:Snn9CG40W-qBMvnr7Zpna,session:0707aeb4-3615-4c54-a72c-20b04f140973]

[7:20:44 AM] [SUCCESS] [🖥️] Host exported with decrypted credentials [op:host_export,user:Snn9CG40W-qBMvnr7Zpna,host:3]

[7:21:53 AM] [ERROR] [🖥️] SSH connection error [op:ssh_connect,host:3]

Error: Keepalive timeout

    at Timeout.sendKA (/app/node_modules/ssh2/lib/client.js:720:23)

    at listOnTimeout (node:internal/timers:585:17)

    at process.processTimers (node:internal/timers:521:7) {

  level: 'client-timeout'

}

[7:21:53 AM] [ERROR] [🖥️] Proceeding with cleanup after error

{

  operation: 'ssh_error_cleanup',


  hostId: 3,

  error: 'Keepalive timeout'

}

[7:21:53 AM] [INFO] [🖥️] Terminal session destroyed [op:session_destroyed,user:Snn9CG40W-qBMvnr7Zpna,host:3,session:8a5bef45-1dfe-4042-b157-fe3e52d6c7ae
```

--- a/src/ui/desktop/apps/features/terminal/Terminal.tsx
+++ b/src/ui/desktop/apps/features/terminal/Terminal.tsx
@@ -1053,7 +1053,7 @@ const TerminalInner = forwardRef<TerminalHandle, SSHTerminalProps>(
         const savedSessionId = persistenceEnabled
           ? localStorage.getItem(`termix_session_${tabId}`)
           : null;
-        if (savedSessionId && !isReconnectingRef.current) {
+        if (savedSessionId) {
           sessionIdRef.current = savedSessionId;
           isAttachingSessionRef.current = true;
 

